packages/powersync_core/pubspec.yaml

```diff
@@ -0,0 +1,43 @@
+name: powersync_core
+version: 0.0.1
```

Comment:
Can we start at a higher version, e.g. 1.0.0? While it's technically a new package, it was based on the powersync one, which is already stable.

Reply:
Yes I will run melos version and bump the version to a stable 1.0.0, once any other issues are resolved.